### PR TITLE
style(kcheckbox): fix label alignment

### PR DESCRIPTION
**Pull request by aanchalm01**
### Summary
Konnect Registration page has Checkbox input/span alignment issue.
#### Changes made:
*

### PR Checklist
- [ ] Does not introduce dependencies
- [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
- [ ] **Tests pass:** check the output of yarn test packages/<Kongponent>
- [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
- [ ] **Framework style:** abides by the essential rules in Vue's style guide
- [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs), or leftover comments
- [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

**Comment by chrisvdp** [comment]
@aanchalm01 looking at the Percy results, the labels are misaligned in Radio docs now. https://deploy-preview-340--kongponents.netlify.app/components/input-radio.html

**Comment by aanchalm01** [comment]
@chrisvdp does the checkbox look okay to you now ? I can do the scoped style to make sure that radio style is not updated.

**Comment by chrisvdp** [comment]
My issue with overly specific CSS rules is that it tends to create a CSS selector arms race. The styles are very hard to override if you need to. This isn't a great approach in websites and it's especially bad approach in style guides as these are consumed by webapps and they are very real use cases where they need to be overridden.

It seems that a [render function](https://vuejs.org/v2/guide/render-function.html) may be needed here to avoid this. A render function should give us access to the raw string so we can trim white space before it makes it to the DOM.

File: docs/components/input-checkbox.md [hunk 1]
```diff
@@ -145,6 +118,15 @@ like:
 .KCheckbox-wrapper {
   --KCheckboxPrimary: blueviolet;
 }
+
+.k-checkbox {
```
Comment (chrisvdp) [inline docs/components/input-checkbox.md:122]: **question:** ‏do we still need this CSS?

**Comment by chrisvdp** [comment]
@aanchalm01 this should be marked as a breaking change

https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with--to-draw-attention-to-breaking-change

File: docs/components/input-checkbox.md (hunk 1 above)
Comment (aanchalm01) [reply docs/components/input-checkbox.md]: No! Removed it. Thanks for catching this.

**Review by chrisvdp — APPROVED** [review]
LGTM

**Pull request merged**